.vscode/settings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "cssCustomProperties.files": ["@navikt/core/css/dist/index.css"],
+  "cssCustomProperties.languages": ["css"],
+  "eslint.problems.shortenToSingleLine": false,
+  "deno.enablePaths": ["scripts/article-views", "scripts/changelog"],
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "deno.lint": true,
+  "search.exclude": {
+    "**/node_modules": true,
+    "**/dist": true,
+    "**/esm": true,
+    "**/cjs": true
+  },
```

Comment:
Tror "search.exclude" er overflødig, da "Use Exclude Settings and Ignore Files" automatisk ignorerer alt i .gitignore.